area.js

```diff
@@ -0,0 +1,8 @@
+function area(rectangle){
+  var width = rectangle.width;
+  var height = rectangle.height;
+  var area = width * height;
+  return area
+};
```

Comment:
I'd suggest just returning `rectangle.width * rectangle.height` and not adding extra bloat. While it may have seemed to add extra readability, I actually think the opposite was true since it's simpler to get to the point for something like `area`.

When we start using classes, defining width and height as attributes of a shape is actually a DRYer practice, but we can discuss more when we get to that.